calblueprint/meepanyar
# Add Formik to Profile pages

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
This PR does the following:
1. adds Formik input validation to the Profile pages. 
2. Adds "Active Status" to `EditUserInformation` to match the Figma

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Log into the app with the following credentials: `username: admin ; password: test`
2. Go to the profile page and the site name and try to break the form by submitting breaking inputs (the only thing I could think of is empty string)
3. Make sure editing a site name properly modifies Airtable
3. Go to an "edit tariff plan" page and try to break the form by submitting negative numbers, empty fields, etc...
4. Make sure editing a tariff plan still properly modifies Airtable.
4. Go to the "User Information" tab and  make sure the "Active Status" button properly modifies Airtable.

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
